src/modules/ie-objects/elasticsearch/templates/exact/identifier-search-query.json

```diff
@@ -26,6 +26,11 @@
           "term": {
             "premis_identifier": "{{query}}"
           }
+        },
+        {
+          "term": {
+            "schema_maintainer_id": "{{query}}"
```

Comment:
does this field exist in elasticsearch?
shouldn't this use the id in the maintainer object? 
schema_maintainer.schema_identifier
```
...
"schema_contributor": null,
"schema_maintainer": {
    "organization_type": "Cultuur",
    "schema_identifier": "OR-154dn75",
    "alt_label": "amsab-isg",
    "schema_name": "Amsab-ISG"
},
"meemoo_local_id": "umatic_107",
...
```